data/curated/american_idol/meta.yaml

```
@@ -0,0 +1,11 @@
+title: American Idol data "This week we're exploring {title}!"
```

```suggestion
title: American Idol data
```
I still need to nail this explanation! This line must ONLY include the {title} piece of that example.